src/Twig/Spell.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace js\tools\numbers2words\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+use js\tools\numbers2words\Speller;
+
+class Spell extends \Twig\Extension\AbstractExtension
```

Comment:
The parent class is already imported...

Reply:
Use the import, Luke!

Reply:
Jurchiks, I just follow the recommended approach - https://symfony.com/doc/current/templating/twig_extension.html

Reply:
I meant `class Spell extends AbstractExtension`.